### DISCUSSION TITLE
请问什么时候支持世纪互联

### DISCUSSION THREAD

**Discussion by ai-yoyo** [spencerwooo/onedrive-vercel-index#71] [Q&A] [answered]
看到CF版本支持，请问这里什么时候会支持。谢谢

**Comment by spencerwooo** [comment, marked as answer]
没测试过，但在 `config/api.json` 里面换上世纪互联相应的 `authApi` 和 `driveApi` 地址或许就可以直接支持了呢～

**Reply by spencerwooo** [reply]
目测应该将 `config/api.json` 里面替换为类似：

```diff
- "authApi": "https://login.microsoftonline.com/common/oauth2/v2.0/token",
+ "authApi": "https://login.chinacloudapi.cn/common/oauth2/v2.0/token",
- "driveApi": "https://graph.microsoft.com/v1.0/me/drive",
+ "driveApi": "https://microsoftgraph.chinacloudapi.cn/v1.0/sites/{hostName}:{sitePath}"
```

其中的：

- hostName: sharepoint site hostname (e.g. 'name.sharepoint.com')
- sitePath: sharepoint site path (e.g. '/sites/name')

**Reply by aidenlx** [reply]
世纪互联的onedrive只需要将`driveApi`设置为`https://microsoftgraph.chinacloudapi.cn/v1.0/me/drive`即可，上述hostName和sitePath仅在SharePoint时需要
